companies/salesforce.json

```diff
@@ -0,0 +1,55 @@
+{
+    "slug": "salesforce",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "telecommunication"
+    ],
+    "name": "Salesforce.com, Inc.",
+    "runs": [
+        "Salesforce Customer360",
+        "Salesforce Marketing Cloud (formerly ExactTarget)",
+        "Salesforce Commerce Cloud (formerly Demandware)",
+        "Salesforce Social Studio (formerly Radian6)",
+        "SalesforceIQ (formerly RelateIQ)",
+        "Heroku",
+        "Quip",
+        "Pardot",
+        "Tableau",
+        "Work.com"
+    ],
+    "address": "415 Mission St\n3rd Floor\nSan Francisco, CA 94105\nUnited States",
+    "phone": "+1 844 287 7147",
+    "email": "privacy@salesforce.com",
```

Comment:
Here you can set `suggested-transport-medium` to `email` as well, because its a privacy-focussed email